src/middleware.js

```diff
@@ -41,7 +41,7 @@ const createIdleMiddleware = ({
   };
 
   const idleInterval = () => {
-    if (interval === null) {
```

Comment:
this works in the browser?

Reply:
Yes, though I would like to know what made you think this would not work in the browser? I may not fully understand your question.